lightstep/provider_test.go

```diff
@@ -24,7 +24,7 @@ func init() {
 
 	testProject = os.Getenv("LIGHTSTEP_PROJECT")
 	if testProject == "" {
-		testProject = "terraform-provider-tests"
+		testProject = "terraform-provider-test"
```

Comment:
thank you! I didn't realize this was already set automatically, I had just made an env variable